gcp_variant_transforms/libs/variant_merge/move_to_calls_strategy.py

```diff
@@ -118,13 +130,14 @@ def get_merged_variants(self, variants, unused_key=None):
     return [merged_variant]
 
   def get_merge_keys(self, variant):
-    yield ':'.join(
+    key = ':'.join(
         [str(x) for x in [
             variant.reference_name or '',
             variant.start or '',
             variant.end or '',
-            variant.reference_bases or '',
-            ','.join(variant.alternate_bases or [])]])
+            hashlib.sha256(variant.reference_bases or '').hexdigest(),
+            hashlib.sha256(','.join(variant.alternate_bases or [])).hexdigest()]])
```

Comment:
Consider making a private method _get_hash_key and use it here.

Reply:
Done!